src/common/components/forms/Textarea.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import { Textarea, TextareaProps } from '@chakra-ui/react';
+
+type CustomTextareaProps = TextareaProps;
+
+export default function CustomTextarea({ ...props }: CustomTextareaProps) {
```

Comment:
It's best to have a field to determine what variant we want to return when using this component.